src/viam/examples/camera/example_camera.cpp

```diff
@@ -18,17 +18,18 @@ int main() {
         // If you want to connect to a remote robot, this should be the url of the robot
         // Ex: xxx.xxx.viam.cloud
         std::string robot_address("localhost:8080");
-        // If you want to connect to a remote robot, you need the robot location secret (not
-        // the part secret) You can find this on app.viam.com
-        vs::Credentials credentials("");
+        // If you want to connect to a remote robot, you need some authentication secret
+        // You can find this on app.viam.com
+        vs::Credentials credentials("fake-credential-type", "");
 
         vs::DialOptions dial_options;
 
         // If you have credentials, use this to pass them to the robot
         // dial_options.credentials = credentials;
 
         // This is for an example. Care should be taken before exercising this option in production.
-        dial_options.set_allow_insecure_downgrade(credentials.payload().empty());
+        dial_options.set_allow_insecure_downgrade(
```

Comment:
I think we're changing the way this works given the code as written. Previously, this would evaluate to true (because the credential payload is empty), but now it evaluates to false (because the credential type is _not_ empty). Probably better to have the credential type we pass be an empty string rather than a fake type. We can always add a comment to clarify what the args are if you think it would be useful.

Reply:
Is this referencing line 23 above?

Reply:
Also in terms of comments I think it might look clogged if we add more and the comments already tell the user to add credentials so I think it is self explanatory that this example is passing empty creds.

Reply:
Yep, line 23. Sounds good!